startup_plugin.sh

```diff
@@ -0,0 +1,7 @@
+vault login root
+vault secrets enable -path=kubernetes-secrets-reader vault-plugin-secrets-kubernetes-reader
+vault policy write read-plugin-secrets - <<EOF
+        path "kubernetes-secrets-reader/*" {
```

Comment:
Is there a way to enable access only to secrets in a specific namespace (e.g., in M4D when not using builtin catalog you only need the manager to access secrets in m4d-system).